week-1/interpret/to-pounds.js

```diff
@@ -1,20 +1,35 @@
-const penceString = "399p";
+const penceString = "0p";
```

Comment:
I'm going to assume this is a testing change! In which case, nice to see that you're trying the code out with edge cases. Before committing, it may help to review all the files that have changed

Reply:
Thanks a lot for your time! Definitely, cheers!